src/xcode/ENA/ENAUITests/AntigenTestProfile/ENAUITests_12_AntigenTestProfile.swift

```diff
@@ -136,10 +140,40 @@ class ENAUITests_12_AntigenTestProfile: CWATestCase {
 		
 		/// Antigen Test Profile Screen
 		
-		// delete profile button exists
-		let deleteTestProfileButton = try XCTUnwrap(app.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Profile.deleteButton])
+		// edit profile button exists
+		var editTestProfileButton = try XCTUnwrap(app.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Profile.editButton])
+		editTestProfileButton.waitAndTap()
+
+		snapshot("antigentestprofile_screenshot_edit_actions")
+
+		/// Edit Antigen Test Profile Screen
+
+		let editTestProfileAction = try XCTUnwrap(app.sheets.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Profile.editAction])
+		editTestProfileAction.waitAndTap()
+
+		let editSaveProfileButton = try XCTUnwrap(app.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Create.saveButton])
+		XCTAssertTrue(editSaveProfileButton.waitForExistence(timeout: .short))
+		XCTAssertTrue(editSaveProfileButton.isEnabled)
```

Comment:
Before changing something in the UI, is the Button enabled? If so, everything fine, if not, should this be `XCTAssertFalse(`?

Reply:
It should be enabled here, because the fields should be filled from the previous steps.